public/friendlycode/js/fc/load-project-files.js

```diff
@@ -0,0 +1,168 @@
+define([], function() {
+  "use strict";
+
+  var Path = Bramble.Filer.Path;
+  var FilerBuffer = Bramble.Filer.Buffer;
+  var self = {};
```

Comment:
This isn't a great pattern.  Don't call this `self`, just have everything be local functions, and then do:

```js
return {
  load: load,
  ...
};
```